packages/trader/src/Modules/CFD/Containers/cfd-dashboard.jsx

```diff
@@ -39,139 +39,128 @@ const LoadTab = ({ children, is_loading, loading_component, ...props }) => {
     return <Tabs {...props}>{children}</Tabs>;
 };
 
-class CFDDashboard extends React.Component {
-    state = {
-        is_demo_enabled: false,
-        is_real_enabled: false,
-        active_index: 0,
-        is_account_needed_modal_open: false,
-        is_demo_tab: true,
-        required_account: {},
-        is_notification_loaded: false,
-        password_manager: {
-            is_visible: false,
-            selected_login: '',
-            selected_account: '',
-            selected_account_type: '',
-            selected_account_group: '',
-            selected_server: '',
-        },
-    };
-
-    async componentDidMount() {
-        this.updateActiveIndex(this.getIndexToSet());
-        this.openResetPassword();
-        this.props.onMount();
-    }
-
-    componentWillUnmount() {
-        this.props.onUnmount();
-    }
-
-    componentDidUpdate() {
-        this.updateActiveIndex();
-        this.props.checkShouldOpenAccount();
-
-        if (this.props.is_logged_in) {
+const CFDDashboard = props => {
+    const [is_demo_enabled, setIsDemoEnabled] = React.useState(false);
+    const [is_real_enabled, setIsRealEnabled] = React.useState(false);
+    const [active_index, setActiveIndex] = React.useState(0);
+    const [is_demo_tab, setIsDemoTab] = React.useState(true);
+    const [is_notification_loaded, setIsNotificationLoaded] = React.useState(false);
+    const [password_manager, setPasswordManager] = React.useState({
```

Comment:
(nitpick) Using `useReducer` makes things easier here or not? :eyes:

Reply:
Since multiple useState is the prefered way for backward compatibility, I think it's better to just stick to the old way.